saleor/graphql/channel/types.py

```diff
@@ -119,11 +123,32 @@ class Meta:
 
 class Channel(ModelObjectType):
     id = graphene.GlobalID(required=True)
-    name = graphene.String(required=True)
-    is_active = graphene.Boolean(required=True)
-    slug = graphene.String(required=True)
-    currency_code = graphene.String(required=True)
     slug = graphene.String(required=True)
+
+    name = PermissionsField(
```

Comment:
We are missing descriptions for these fields, it's a good occasion to add them ;)